modules/shared/components/molecules/UserInfo/Index.tsx

```diff
@@ -11,14 +15,27 @@ const UserInfo: FC<IUserInfo.IProps> = ({
   name,
   date,
 }): ReactElement => {
+  const [isShown, setIsShown] = useState(false);
   return (
     <div className={styles['outer-wrapper']}>
       <Avatar size={size} variant={variant} imgSrc={imgSrc} />
       <div className={styles['user-wrapper']}>
         <span className={styles.name} data-testid="name">
           {variant === 'anonymous' ? 'Anonymous' : name}
         </span>
-        <span className={styles.date}>{date} ago</span>
+        <div
```

Comment:
dont complete the logic here. use the title attribute to show the full datetime upon hovering the span.

Reply:
So, the task is not to replace the (date from now) altogether with the (full DateTime) upon hovering? if i get it right!
rather is to just use the title attribute which is by default upon hovering shows itself in a little nice box. is this what i have got to do?

Reply:
sorry. my auto correction is so bad. I meant don't "complicate" not "complete".

and yes. you are absolutely correct and thanks for reiterate over and ensure that you got the requirement correctly. I noticed that you keep doing this to avoid any miscommunication which is something I highly appreciate and encourage everyone to do the same 😍